src/components/overlay/index.js

```diff
@@ -50,35 +50,37 @@ export default class Overlay extends PureBaseComponent {
 
   renderCustomContent = () => {
     const {customContent} = this.props;
-    return <View style={styles.customContent}>{customContent()}</View>;
-  };
-
-  renderImage(typeStyle) {
-    const {type, customContent} = this.props;
-    const image = type !== OVERLY_TYPES.SOLID ? gradientImage : undefined;
-
     return (
-      <View style={styles.container}>
-        {type && <Image style={[styles.container, typeStyle]} resizeMode={'stretch'} source={image}/>}
-        {customContent && this.renderCustomContent()}
+      <View pointerEvents="none" style={styles.customContent}>
+        {customContent}
       </View>
     );
-  }
+  };
+
+  renderImage = (style, source) => {
+    return <Image style={[styles.container, style]} resizeMode={'stretch'} source={source}/>;
+  };
 
   render() {
     const {type, customContent} = this.props;
+    const image = type !== OVERLY_TYPES.SOLID ? gradientImage : undefined;
 
     if (type === OVERLY_TYPES.VERTICAL) {
       return (
-        <View style={[styles.container]}>
```

Comment:
Consider using Fragment (`<>`) instead of the container View for both Vertical case and the other case